docs/tasks/administer-cluster/configure-pod-disruption-budget.md

```diff
@@ -55,9 +81,16 @@ specified in the budget, thus bringing the number of available pods from the
 collection below the specified size. The budget can only protect against
 voluntary evictions, not all causes of unavailability.
 
-You can find an example of a pod disruption budget defined below. It matches pods with the label 
+A `maxUnavailable` of 0% (or 0) or a `minAvailable` of 100% (or equal to the
+number of replicas) may block node drains entirely. This is permitted as per the 
+semantics of `PodDisruptionBudget`. Cluster administrators may restrict this 
+behavior through admission control.
```

Comment:
Remove sentence "Cluster administrators may restrict this 
 +behavior through admission control.".

They can't do this without recompiling or a lot of effort, so that isn't true.  Also, restricting this doesn't prevent conditions where draining gets blocked.